offix-offline/src/main/java/org/aerogear/offixOffline/CacheResolver.kt

```diff
@@ -0,0 +1,35 @@
+package org.aerogear.offixOffline
+
+import android.util.Log
+import com.apollographql.apollo.api.Operation
+import com.apollographql.apollo.api.ResponseField
+import com.apollographql.apollo.cache.normalized.CacheKey
+import com.apollographql.apollo.cache.normalized.CacheKeyResolver
+
+class CacheResolver {
+
+    private val TAG = javaClass.simpleName
+
+    fun cacheResolver(): CacheKeyResolver {
+        return object : CacheKeyResolver() {
+            override fun fromFieldRecordSet(field: ResponseField, recordSet: Map<String, Any>): CacheKey {
+                Log.d(TAG, "fromFieldRecordSet ${(recordSet["id"] as String)}")
```

Comment:
Also, why did we hardcode the id here?
Is it something that's mandatory for all the type of mutations that the user will ever create?

Reply:
ID: The ID scalar type (default in graphQL) represents a unique identifier, often used to refetch an object or as the key for a cache. The ID type is serialized in the same way as a String; however, defining it as an ID signifies that it is not intended to be human‐readable.